src/sshfs_mount/sshfs_mounts.cpp

```diff
@@ -65,6 +67,12 @@ mp::SSHFSMounts::SSHFSMounts(const SSHKeyProvider& key_provider) : key(key_provi
 void mp::SSHFSMounts::start_mount(VirtualMachine* vm, const std::string& source_path, const std::string& target_path,
                                   const mp::id_mappings& gid_mappings, const mp::id_mappings& uid_mappings)
 {
+    if (!MP_FILEOPS.exists(QDir{QString::fromStdString(source_path)}))
+    {
+        mount_processes[vm->vm_name].erase(target_path);
+        throw std::runtime_error(fmt::format("Mount path `{}` does not exist.", source_path));
```

Comment:
i see that you're using backticks here, but in most other places single quotes are used for printing paths